doc/Command-Reference.md

```diff
@@ -10397,6 +10397,38 @@ This command removes all unused images from the device, leaving only the current
   No image(s) to remove
   ```
 
+**sonic-install get-fips**
+
+This command is used to get the fips status.
+
+- Usage:
+  ```
+  sonic-installer get-fips [<image_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-installer get-fips
+  FIPS is enabled
+  ```
+
+**sonic-install set-fips**
```

Comment:
```suggestion
**sonic-installer set-fips**
```
Another one.

Reply:
> 



> Another one.

@mdanish-kh , thanks for your review, fixed.